pages/community/styleguide/code-examples.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Code examples
+weight: 20
+---
+
+# Code Examples
+
+## Guidelines
+
+Code examples which you add must be simple enough to understand and with good details to do something interesting and useful.
+
+You need to add practical code examples which adhere to industry standards as users will copy the sample code and run it in production. Be sure to include any warnings through comments which might occur during the code run.
+Make it clear if it is a snippet and not a complete example. Provide information about any packages or dependencies required to run the sample code.
+
+The code example should be self explanatory and easy to understand. The goal is to provide simplified working examples that can be understood as soon as possible, not necessarily to create efficient, brilliant code that impresses experts and has outstanding utility.
+
+
+## Formatting 
+
+### Code Length
+
+- Code lines shouldn't be so lengthy that reading them horizontally requires scrolling.
+
+- For the sake of readability, break large lines at appropriate moments, but never at the price of acceptable practises.
+
+- Add bullet points for better readability and understandability.
```

Comment:
add bullet points in code examples? what?

I am not sure this sentence makes sense 😅 

```suggestion
```

Reply:
Here, if we are explaining that code example then for better understanding we can add bullet points

Reply:
No, we don't need to use bullet points in code snippet examples. **Please delete this sentence, because it is not relevant to code snippets.** (bullet points would be text written around code snippets, they are not a part of the code blocks)